src/campaign.ts

```diff
@@ -65,9 +65,9 @@ export class Campaign<StreamChatGenerics extends ExtendableGenerics = DefaultGen
     return this.client.stopCampaign(this.id as string);
   }
 
-  async get() {
+  async get(users?: { limit?: number; next?: string; prev?: string }) {
```

Comment:
@kanat is it possible that more arguments would be added in the future? If there is 0.0000001% of posibility this will happen, it would be better to keep a single method argument that would be an object:

```suggestion
  async get({users}: GetCampaignOptions) {
```

and the type definition could be:

```
type GetCampaignOptions = {
  users?: { limit?: number; next?: string; prev?: string }
};
```